c_cpp_style.md

```diff
@@ -312,20 +312,17 @@ literal and explicit cast must be used for constants.
 >     .1 /* no! */
 >     .1f /* no! */
 
-- In the function definition, the return type of a function should be placed in a different line of the function.
+- In the function definition, the return type of a function should be placed in the same line of the function.
 - The function parameters should be on the same function line if they fit; otherwise, wrap arguments at the parenthesis.
 
->     int32_t
->     ComputeActualVelocity(int32_t actualPosition, int32_t lastPosition);
+>     int32_t   ComputeActualVelocity(int32_t actualPosition, int32_t lastPosition);
```

Comment:
Could you add that if it's static function this start without caps computeActualVelocity, and if it's public the name of the module in caps underscore, and no caps. PROFILE_computePositionProfile()?

Reply:
Hecho.